Source/ParameterEncoding.swift

```diff
@@ -261,6 +293,24 @@ public struct URLEncoding: ParameterEncoding {
             return false
         }
     }
+
+    private func encodeArrayKey(_ key: String) -> String {
```

Comment:
It would be cleaner if this was simply an `encode` function on `ArrayEncoding`. Better encapsulation that way.